src/lib/render-lit-html.ts

```diff
@@ -374,6 +374,29 @@ export async function* renderTemplateResult(
               if (reflectedName !== undefined) {
                 yield `${reflectedName}="${value}"`;
               }
+            } else if (attr.name.startsWith('@')) {
+              // Event binding
+              // do nothing with values
+              partIndex += statics.length - 1;
+            } else if (attr.name.startsWith('?')) {
+              // Boolean attribute binding
+              const attributeName = attr.name.substring(
```

Comment:
More concise: `attr.name.slice(1, -5)`

More semantic: `attr.name.slice(1, -boundAttrSuffix.length)`

Same for L352, similar for L401

Related (and noted in another PR I think?), but L339 should use `boundAttributeSuffix`.

Reply:
done